node/src/components/small_network.rs

```diff
@@ -575,10 +575,13 @@ where
         add_to_blocklist: bool,
     ) -> Effects<Event<P>> {
         if let Some(incoming) = self.incoming.remove(&peer_id) {
+            trace!(%peer_id, "removing peer from the incoming connections");
             let _ = self.pending.remove(&incoming.peer_address);
         }
         if let Some(outgoing) = self.outgoing.remove(&peer_id) {
+            trace!(%peer_id, "removing peer from the outgoing connections");
             if add_to_blocklist {
+                trace!(%peer_id, "blacklisting peer");
```

Comment:
I think this should be at `info` level so it actually appears in the users' logs.

Reply:
Done in [`af4ffe3` (#738)](https://github.com/CasperLabs/casper-node/pull/738/commits/af4ffe3d128db7e6ca26bdb73da6cc90cfb79a27)